addons/calendar_button/class/Calendar.gd

```diff
@@ -68,7 +68,7 @@ func get_number_of_days(m, y):
 # Returns the weekday (int)
 func get_weekday(d, m, y):
 	var t = [0, 3, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4]
-	y -= m < 3
+	if m < 3: y -= m
```

Comment:
Hi, 
Sorry for reviving a dead thread but I might have found a problem with this line.
Shouldn't it be: `if m < 3: y -= 1`

Reply:
You are right! I had already discovered this bug and fixed it, but I didn't have committed the change yet.
Thanks
-j

Reply:
just updated my fork repo!

Reply:
I am very sorry, but could you merge your latest changes with master and reattempt the PR?

I have been away for quite some time, so I have failed to follow-up on this PR. 
The latest version should work as expected, if you made any unique changes/feature improvements, please highlight them :)